### PR TITLE
support direct source loading like a webpack loader

### PR DESCRIPTION
**Pull request by aleen42**
How about exposing a method of the tool itself, where developers can load source directly and act like a webpack loader?

### CODE REVIEW

**Comment by aleen42** [comment]
However, this tool is mainly used under node environment rather than the browser. Such a PR may be taken more considerations.

**Comment by oscarotero** [comment]
I'm not sure about the worth of include sketch files in webpack, but is a good idea to have a way to load a file from its content.

File: index.js [hunk 1]
```diff
@@ -4,36 +4,19 @@ const Sketch = require('./src/Sketch');
...
      * @return {Promise}
      */
-    lib.read = function(file) {
-        if (Array.isArray(file)) {
-            return Promise.all(file.map(each => lib.read(each)));
-        }
-
-        return JSZip.loadAsync(fs.readFileSync(file))
+    lib.load = function(source) {
+        JSZip.loadAsync(source)
```
[the remark this answers is not in the document]
Comment (aleen42) [reply index.js]: Sorry for the mistake.

**Comment by oscarotero** [comment]
Thanks!

**Pull request merged**